5.application_scopes.md

```diff
@@ -19,44 +20,36 @@ Components A, B, and C are deployed to the same health scope. The health scope w
 
 Component A is isolated in its own network scope from components B, C, and D. This allows the infrastructure operator to supply different SDN settings for different groups of components, such as more restricted inbound/outbound rules on back-end components.
 
-## Application scope types
-There are two kinds of application scope types:
- - Core application scope types
- - Extended application scope types
-
-### Core application scope types
-Core application scope types define grouping constructs for basic runtime behavior. They have the following characteristics:
+## Categories of application scope
 
- - Core application scope types MUST be in the `core.oam.dev` namespace.
- - Core application scope types MUST all be supported by any implementation of the specification. They may be implemented by any available feature in the platform, but they MUST be implemented according to the specification of each core application scope type.
- - Instances of core *workload* types MUST be deployed into an instance of each core application scope type. 
- - Runtimes MUST provide a default "root" application scope instance for each core application scope type.
- - Runtimes MUST deploy each component instances into default "root" application scope instances when an application configuration does not otherwise specify an application scope for a component instance.
+There are two kinds of application scope types:
 
- The following core application scope types are defined by this specification:
+ - **Core application scope types**: Core application scope types define grouping constructs for basic runtime behavior. They have the following characteristics:
+    - Core application scope types MUST be in the `core.oam.dev` namespace.
+    - Core application scope types MUST all be supported by any implementation of the specification. They may be implemented by any available feature in the platform, but they MUST be implemented according to the specification of each core application scope type.
+    - Instances of core *workload* types MUST be deployed into an instance of each core application scope type. 
```

Comment:
By "Instances of core *workload* types" are we referring to the workload CRs? Or are we referring to components that are configured to a certain kind of workload?

Reply:
It means workload CRs, I should fix it

Reply:
lgtm